### PR TITLE
Add partner logos to "Chat with us" page

### PR DESCRIPTION
**Pull request by jenny-heath**
This includes adding the vita partner model, which puts the code at a part-way point to the new method of routing

Co-authored-by: Ben Vandgrift <bvandgrift@codeforamerica.org>

### CODE REVIEW

File: db/migrate/20200429190910_add_partner_information_to_intake.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class AddPartnerInformationToIntake < ActiveRecord::Migration[6.0]
+  def change
+    add_column :intakes, :vita_partner_name, :string
+
+    reversible do |dir|
```
Comment (jenny-heath) [inline db/migrate/20200429190910_add_partner_information_to_intake.rb:5]: interesting - is this a backfill combined with a migration? never seen anything like this

File: app/models/intake.rb [hunk 2]
```diff
@@ -396,6 +410,21 @@ def filing_years
...
 
+  def assign_vita_partner!
+    # NOTE: this MUST be called before create_intake_ticket is called.
+    return if vita_partner.present?
+    # given the information on the intake
+    get_or_create_zendesk_group_id
+    get_or_create_zendesk_instance_domain
+    if zendesk_group_id
+      # this will not be true for intakes to the UWTSA instance
+      # TODO: log this!
```
Comment (jenny-heath) [inline app/models/intake.rb:421]: what are we logging?

File: app/models/vita_partner.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: vita_partners
+#
+#  id                      :bigint           not null, primary key
+#  display_name            :string
+#  drop_off_code           :string
+#  logo_url                :string
+#  name                    :string           not null
+#  referral_code           :string
```
Comment (bengolder) [inline app/models/vita_partner.rb:10]: might be helpful to rename this to `source_parameter_prefix` to make it consistent with other language throughout the code.

File: db/migrate/20200428220200_create_vita_partners.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+class CreateVitaPartners < ActiveRecord::Migration[6.0]
+  def change
+    create_table :vita_partners do |t|
+      t.string :name, null: false
+      t.string :display_name
+      t.string :logo_url
```
Comment (bengolder) [inline db/migrate/20200428220200_create_vita_partners.rb:6]: might make sense to call this `logo_filename` or `logo_path` if it's just storing the name of the image file. I would generally expect a _url property to be a full URL

**Pull request merged**